README.md

```diff
@@ -25,10 +25,15 @@ for Google Cloud Platform :
 
 * Instantiate an image with the resulting AMI
   * Make sure it has a public IP address
-  * Make sure it is in a security group with the following inbound rules:
+  * Make sure it is in a security group with at least the following inbound rules:
     * UDP 500
     * UDP 4500
     * TCP 22 to your IP (for SSH management)
+  * Make sure it is in a security group with at least the following outbound rules:
+    * UDP 500
+    * UDP 4500
+    * TCP 1024-65535 to your IP (for SSH management)
```

Comment:
I don't think this is necessary, but we should test this. Some firewalls are going to allow port 22 outbound if the TCP session was allowed as part of an inbound rule.

Reply:
Need this open to ssh in to the VPN.  Client connects to the VPN on port 22 (inbound) and the VPN connects to the client on the port the client provided which is between 1024-65535 (outbound).